Student_BCOs/LingSheng_Loh_BCO_000529.txt

```diff
@@ -1,22 +1,58 @@
 {
     "bco_id": "http://biocomputeobject.org/BCO_000529",
-    "checksum": "cc6b375559be5e90ed1162a4a15ebb65e992b100479c53d63cdb36a060ccade0",
+    "checksum": "a4416da0f8797b81b6bedc008062e122bdbf2b8d4d995db5f6d12ba8a5d37178",
     "bco_spec_version": "1.3.0",
     "provenance_domain": {
         "name": "Correlating miRNA and gene expression data in 34 cancer phenotypes",
-        "version": "1.0",
+        "version": "1.0.0",
         "license": "Creative Commons Attribution 4.0 International License",
         "created": "2019-10-08T02:27:36.285940",
-        "modified": "2019-10-08T21:11:58.377902",
+        "modified": "2019-10-16T01:10:32.038945",
+        "review": [
+            {
+                "status": "in-review",
+                "reviewer_comment": "Reviewed by BioCompute technical assistant, approval pending modification by author",
+                "reviewer": {
+                    "orcid": "https://orcid.org/0000-0002-8824-4637",
+                    "affiliation": "George Washington University",
+                    "contribution": [
+                        "curatedBy"
+                    ],
+                    "name": "Janisha Patel",
+                    "email": "janishapatel@gwu.edu"
+                },
+                "date": "2019-10-15"
+            }
+        ],
         "contributors": [
             {
                 "name": "Ling Sheng Loh",
-                "orcid": "",
-                "affiliation": "",
+                "orcid": "https://orcid.org/0000-0003-0981-7984",
+                "affiliation": "George Washington University",
                 "contribution": [
                     "createdBy"
                 ],
                 "email": "loh_ling_sheng@hotmail.com"
+            },
+            {
+                "name": "Micah Madrid",
+                "orcid": "https://orcid.org/0000-0002-7527-022X",
+                "affiliation": "George Washington University",
+                "contribution": [
+                    "authoredBy",
+                    "derivedFrom"
+                ],
+                "email": "mmadrid@gwu.edu"
+            },
+            {
+                "name": "June Beshea",
+                "orcid": "https://orcid.org/0000-0001-9101-8000",
+                "affiliation": "George Washington University",
+                "contribution": [
+                    "authoredBy",
+                    "derivedFrom"
+                ],
+                "email": "jbeshea@gwu.edu"
```

Comment:
Please remove your team members from the contributor list and add the publication authors to the list. See #112